waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -214,17 +214,12 @@ genServerJs spec =
       (C.asServerFile [relfile|src/server.ts|])
       ( Just $
           object
-            [ "doesServerSetupFnExist" .= isJust maybeSetupJsFunction,
-              "serverSetupJsFnImportStatement" .= fromMaybe "" maybeSetupJsFnImportStmt,
-              "serverSetupJsFnIdentifier" .= fromMaybe "" maybeSetupJsFnImportIdentifier,
+            [ "setupFn" .= extImportToImportJson relPathToServerSrcDir maybeSetupJsFunction,
```

Comment:
Simplification by using `extImportToImportJson`, we did this convetions for web app imports and doing it for server code now.